utils/auto_annotation/README.md

```diff
@@ -50,3 +50,16 @@ $ python cvat/utils/auto_annotation/run_modely.py --py /path/to/python/interp.py
                                                   --show-images
                                                   --show-image-delay 2000
 ```
+
+Visualization isn't always enough. The CVAT has a serialization step that can throw confusing errors on model upload.
+You must install the necessary packages installed, but then you can add the `--serialize` command to ensure that your 
+results will serialize correctly.
+
+```shell
+$ python cvat/utils/auto_annotation/run_modely.py --py /path/to/python/interp.py \
```

Comment:
Probably typos is here: run_model``y``.py

Reply:
@bsekachev nice catch, fixed!